cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -480,6 +480,50 @@ a|
 Terminate transaction now allows `YIELD`.
 The `WHERE` clause is not allowed on its own, as it is for `SHOW`, but needs the `YIELD` clause.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS [transaction-id[,...]]
+----
+[source, cypher, role="noheader"]
+----
+TERMINATE TRANSACTIONS transaction-id[,...]
+----
+a|
+The `transaction-id` now allows general expressions resolving to a string or a list of strings instead of just parameters.
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS [transaction-id[,...]]
+YIELD field[, ...]
+  [ORDER BY field[, ...]]
+  [SKIP n]
+  [LIMIT n]
+  [WHERE expression]
+TERMINATE TRANSACTIONS transaction-id[,...]
+YIELD field[, ...]
+  [ORDER BY field[, ...]]
+  [SKIP n]
+  [LIMIT n]
+  [WHERE expression]
+RETURN field[, ...]
+  [ORDER BY field[, ...]]
+  [SKIP n]
+  [LIMIT n]
+----
+a|
+The `SHOW` and `TERMINATE TRANSACTIONS` commands can be combined in the same query.
+The query does not require a specific order and there can be zero or more of each command type, however at least one command is needed.
+
+When the command is not in standalone mode, the `YIELD` and `RETURN` are mandatory and `YIELD *` is not allowed.
+
+The `transaction-id` is a comma-separated list of one or more quoted strings or an expression resolving to a string or a list of strings (for example the output column from `SHOW`).
```

Comment:
```suggestion
The `transaction-id` is a comma-separated list of one or more quoted strings.
It could also be an expression resolving to a string or a list of strings (for example the output column from `SHOW`).
```